.gitignore

```diff
@@ -31,5 +31,7 @@ tmp
 /catalog/kuadrant-operator-catalog.Dockerfile
 /coverage/
 
+/kuadrant-ratelimit-wasm
```

Comment:
maybe call it `kuadrant-ratelimit-shim` ?

Reply:
it will be `kuadrant-wasm-shim`